src/src_main/crypto_key_mgmt.c

```diff
@@ -81,7 +83,7 @@ int32_t Crypto_Key_OTAR(void)
     }
 
     count = sdls_frame.pdu.pdu_len - MAC_SIZE;
-    for (int w = 0; w < 16; w++)
+    for (w = 0; w < 16; w++)
```

Comment:
I disagree with pulling out the declaration like this 👎  ... works though /shrug.